crates/ruff_server/docs/setup/HELIX.md

```diff
@@ -52,10 +52,48 @@ You can pass settings into `ruff server` using `[language-server.ruff.config.set
 
 ```toml
 [language-server.ruff.config.settings]
-line-length = 80
+lineLength = 80
 [language-server.ruff.config.settings.lint]
 select = ["E4", "E7"]
 preview = false
 [language-server.ruff.config.settings.format]
 preview = true
 ```
+
+By default, Ruff does not log anything to Helix. To enable logging, set the `RUFF_TRACE` environment variable
+to either `messages` or `verbose`. This will set the trace level for this server.
+
+```toml
+[language-server.ruff]
+command = "ruff"
+args = ["server", "--preview"]
+environment = { "RUFF_TRACE" = "messages" }
+```
+
+> \[!NOTE\]
+> `RUFF_TRACE=verbose` does not enable Helix's verbose mode by itself. You'll need to run Helix with `-v` for verbose logging.
+
+To change the log level for Ruff (which is `info` by default), use the `logLevel` setting:
+
+```toml
+[language-server.ruff]
+command = "ruff"
+args = ["server", "--preview"]
+environment = { "RUFF_TRACE" = "messages" }
+
+[language-server.ruff.config.settings]
+logLevel = "debug"
+```
+
+You can also divert Ruff's logs to a separate file with the `logFile` setting:
+
+```toml
+[language-server.ruff]
+command = "ruff"
+args = ["server", "--preview"]
+environment = { "RUFF_TRACE" = "messages" }
+
+[language-server.ruff.config.settings]
+logLevel = "debug"
+logFile = "your/log/file/path/log.txt"
```

Comment:
nit: it might be useful to provide a path suggestion here to the directory for Helix's log file. I'm not sure whether Helix expands the `~` (tilde) character to the home directory, you might want to confirm this if you accept this suggestion

```suggestion
logFile = "~/.cache/helix/ruff.log"
```

Reply:
I've tested this, and we don't expand `~` at the moment. I'll make a follow-up issue for this.